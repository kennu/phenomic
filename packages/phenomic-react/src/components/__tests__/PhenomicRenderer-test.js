const PhenomicRenderer = require("../Renderer")
const React = require("react")
const renderer = require("react-test-renderer")

describe("PhenomicRenderer", () => {

  it("should render", () => {
    const tree = renderer.create(<PhenomicRenderer />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("should render with children", () => {
    const tree = renderer.create(
      <PhenomicRenderer>
        <div />
        <div />
      </PhenomicRenderer>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})
