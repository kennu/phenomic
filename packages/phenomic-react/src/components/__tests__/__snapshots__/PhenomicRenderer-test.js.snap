exports[`PhenomicRenderer should render 1`] = `<div />`;

exports[`PhenomicRenderer should render with children 1`] = `
<div>
  <div />
  <div />
</div>
`;
